src/lib/ScannerCommand.ts

```diff
@@ -1,6 +1,10 @@
 import {SfdxCommand} from '@salesforce/command';
 import {FilterType, RuleFilter} from './RuleFilter';
 import {uxEvents} from './ScannerEvents';
+import {initContainer} from '../ioc.config';
+
+// Bootstrap the ioc container
+initContainer();
```

Comment:
The container needs a central place that configures it, this is a single code path that is always hit.